test/index_spec.ts

```diff
@@ -416,64 +413,6 @@ describe('Linkurious class', () => {
     });
   });
 
-  describe('getEdgeTypes method', function() {
```

Comment:
why remove these tests ?

Reply:
we decided to not write tests anymore in the rest client except for rest client specific features (converting LkNode to Ogma node).

instead we will use the rest client directly to invoke API in the server tests

Reply:
don't you think we should at least check that we hit the right api and get the expected response, I think the objectives of the tests are different here, [no strong opinion about it]

Reply:
If we include the rest-client as part of the integration tests, we are going to test that we hit the right API and get the response.

I would like to avoid writing duplicate tests. Unsure what else to test specifically here.